apps/api/.env.example

```
@@ -0,0 +1,13 @@
+# Required
+DATABASE_URL="postgresql://USER:PASSWORD@localhost:5432/your_database_name?schema=public"
+JWT_SECRET="replace-with-a-strong-random-secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)